src/markdown-pages/install-with-kurl/system-requirements.md

```diff
@@ -16,43 +16,66 @@ title: "System Requirements"
-| kubelet     | /var/lib/kubelet/    | 30 GiB and less than 80% full. See [Host Preflights](/docs/install-with-kurl/host-preflights). |
+| Name           | Location             | Requirements                 | Description |
+| -------------- | -------------------- | ---------------------------- | ----------- |
+| etcd           | /var/lib/etcd/       | 2 GB                         | Kubernetes etcd cluster data directory. See the [etcd documentation](https://etcd.io/docs/v3.5/op-guide/hardware/#disks) and [Cloud Disk Performance](#cloud-disk-performance) for more information and recommendations. |
+| kubelet        | /var/lib/kubelet/    | 30 GB *                      | Used for local disk volumes, emptyDir, log storage, and more. See the Kubernetes [Resource Management for Pods and Containers documentation](https://kubernetes.io/docs/concepts/configuration/manage-resources-containers/#local-ephemeral-storage) for more information.  |
+| containerd     | /var/lib/containerd/ | 30 GB *                      | Snapshots, content, metadata for containers and image, as well as any plugin data will be kept in this location. See the [containerd documentation](https://github.com/containerd/containerd/blob/main/docs/ops.md#base-configuration) for more information. |
+| kube-apiserver | /var/log/apiserver/  | 1 GB                         | Kubernetes audit logs. See Kubernetes [Auditing documentation](https://kubernetes.io/docs/tasks/debug/debug-cluster/audit/) for mode information. |
+| kURL           | /var/lib/kurl/       | 10 GB **                     | kURL data directory used to store utilities, system packages, and container images. This directory can be overridden with the flag `kurl-install-directory` (see [kURL Advanced Install Options](/docs/install-with-kurl/advanced-options)) |
+
+*&ast; This requirement depends on the size of the container images and the amout of ephemeral data used by your application containers.*
+
+*&ast;&ast; This requirement can vary depending on your choice of kURL add-ons and can grow over time.*
 
-## Add-on Directory Disk Space Requirements
+In addition to the storage requirements, the Kubernetes [garbage collection](https://kubernetes.io/docs/concepts/architecture/garbage-collection/) process attempts to ensure that the Node and Image filesystems do not reach their minimum available disk space [thresholds](https://kubernetes.io/docs/concepts/scheduling-eviction/node-pressure-eviction/#hard-eviction-thresholds) of 10% and 15% reqpectively.
+For this reason, kURL recommends an additional 20% overhead on top of these disk space requirements for the volume or volumes containing the directories /var/lib/kubelet/ and /var/lib/containerd/.
```

Comment:
Are we sure 20% is enough. On a recent ticket to be safe I've said as much as 2-3x more. Have no problems with it as long we're confident.

Reply:
the other thing is 30gb in var lib kubelet and containerd is arbitrary and also includes padding